docs/release-notes/8437-auth-refactor.rst

```diff
@@ -0,0 +1,17 @@
+:orphan:
+
+**Removed Features**
+
+-  **Breaking Change** We removed the accidentally-exposed Session object from the
+   ``determined.experimental.client`` namespace. It was never meant to be a public API and it was
+   not documented in :ref:`python-sdk`, but it was exposed in that namespace nonetheless. It was
+   also available as a deprecated legacy alias, ``determined.experimental.Session``. It is expected
+   that most users use the Python SDK normally and are unaffected by this change.
+
```

Comment:
-  **Breaking Change** Removed the accidentally-exposed Session object from the ``determined.experimental.client`` namespace. This object was never meant to be a public API and was
   not documented in :ref:`python-sdk`, but was nonetheless exposed in that namespace. It was
   also available as a deprecated legacy alias, ``determined.experimental.Session``. It is expected
   that most users use the Python SDK normally and are unaffected by this change.

Reply:
changed #8437 accordingly